jetstream/export_json.py

```diff
@@ -142,3 +146,82 @@ def export_statistics_tables(
             _export_table(
                 bigquery_client, project_id, dataset_id, table, bucket, target_path, storage_client
             )
+
+
+def _get_experiment_logs_as_json(
+    client: bigquery.Client,
+    dataset: str,
+    table: str,
+    experiment_slug: str,
+    min_timestamp: datetime = None,
+):
+    """Retrieve records from a single table as JSON."""
+
+    query_text = f"""
+        SELECT *
+        FROM {dataset}.{table}
+        WHERE experiment = '{experiment_slug}'
+    """
+    if min_timestamp is not None:
+        query_text += f" AND timestamp >= '{min_timestamp}'"
+
+    query_text += " ORDER BY timestamp ASC"
+
+    job = client.query(query_text)
```

Comment:
The [docs](https://cloud.google.com/bigquery/docs/writing-results) suggest to call `result()` to wait for the query results
```suggestion
    job = client.query(query_text).result()
```

Reply:
Thanks! Added as new commit so I could also update the var name to make sense with this